examples/tokens/ERC1155ownable.vy

```diff
@@ -85,6 +86,11 @@ event URI:
     value: String[MAX_URI_LENGTH]
     id: uint256
 
+event evtContractURI:
```

Comment:
The event for when a basic URI change is made would be just this. 

```suggestion
event URI:
```

It states that if there is a dynamic URI, it's not strictly necessary to include this event. Curiously, the event definition for `URI` is not included.

Reply:
Removed the redundant evtContractURI
changing the contract-level metadata URI will now use the regular URI event